libs/api/src/lib/graphql/user/user.public-mutation.ts

```diff
@@ -148,9 +148,15 @@ export const updatePublicUser = async (
       name,
       firstName,
       preferredName,
-      address: {
-        update: address
-      }
+      address: address
```

Comment:
in case a new user has no address, it would throw an error when trying to update a non-existing address database entry.